packages/pn-personafisica-webapp/src/components/Contacts/DigitalContactsCodeVerification.context.tsx

```diff
@@ -378,7 +372,7 @@ const DigitalContactsCodeVerificationProvider: FC<{ children?: ReactNode }> = ({
           errorMessage={errorMessage?.content}
         />
       )}
-      <Dialog
+      <PnDialog
```

Comment:
Checks the comments left on CodeModal about distance between Title and Content, and Content and Actions